src/aws-cpp-sdk-transfer/include/aws/transfer/TransferManager.h

```diff
@@ -116,6 +117,13 @@ namespace Aws
              * key/val of map entries will be key/val of query strings.
              */
             Aws::Map<Aws::String, Aws::String> customizedAccessLogTag;
+
+            /**
+             * Set the Checksum Algorithm for the transfer manager to use for multipart
+             * upload. Defaults to CRC32. Will be overwritten to use MD5 if computeContentMD5
+             * is set to true.
+             */
+            Aws::S3::Model::ChecksumAlgorithm checksumAlgorithm = S3::Model::ChecksumAlgorithm::CRC32;
```

Comment:
you can set it to none to disable checksum?

Reply:
Yeah there is specific logic for that where we dont